components/processor/gcomapiprocessor/README.md

```diff
@@ -20,7 +20,8 @@ See the [design doc] that explains how this component is going to be used.
        cache is completely refreshed.
     - `incremental_refresh_duration` (default 5m): Duration until instance
        cache is updated with changes.
-
+    - `grafana_cluster_filters` Comma-separated list of cluster ids where grafana instances are hosted
```

Comment:
Maybe expand on it just a little bit by explaining how they can be looked up and why we need those. Otherwise, lgtm

Reply:
added more details